From b9ba80a22bd4ac3438ad38819cbe550ec9e8a3f8 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 16 Dec 2019 13:59:47 +0100
Subject: [PATCH] po/LINGUAS: drop the polish translation

There are no actual translated strings, and newer gettext (0.20)
fails in this case.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 po/LINGUAS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/po/LINGUAS b/po/LINGUAS
index 6f927b706..aa01eb56e 100644
--- a/po/LINGUAS
+++ b/po/LINGUAS
@@ -1,5 +1,4 @@
 en
-pl
 fr
 cs
 
-- 
2.17.1

